std/typecons.d

```diff
@@ -3966,6 +3986,63 @@ if (is(Interface == interface) && is(BaseClass == class))
     mixin(autoImplement_helper_.code);
 }
 
+///
+@system unittest
+{
+    interface PackageSupplier
+    {
+        int foo();
+        int bar();
+    }
+
+    static abstract class AbstractFallbackPackageSupplier : PackageSupplier
+    {
+        protected PackageSupplier default_, fallback;
+
+        this(PackageSupplier default_, PackageSupplier fallback)
+        {
+            this.default_ = default_;
+            this.fallback = fallback;
+        }
+
+         // Workaround https://issues.dlang.org/show_bug.cgi?id=2525
```

Comment:
I hate to reference a bug in public facing documentation. Who will remember to remove this when the bug is fixed?

Reply:
Me too, but I thought we somehow should point out that these redundancy isn't on purpose -> removed then.